src/dbnode/persist/fs/merger_test.go

```diff
@@ -463,8 +467,9 @@ func testMergeWith(
 		BlockStart: startTime,
 	}
 	mergeWith := mockMergeWithFromData(t, ctrl, diskData, mergeTargetData)
-	err := merger.Merge(fsID, mergeWith, 1, preparer, nsCtx, &persist.NoOpColdFlushNamespace{})
+	prepared, err := merger.Merge(fsID, mergeWith, 1, preparer, nsCtx, &persist.NoOpColdFlushNamespace{})
 	require.NoError(t, err)
+	require.NoError(t, prepared.Close())
```

Comment:
nit: worth adding a tracker to the test DeferClose  method to verify it's called correctly?